### PR TITLE
(baby-project-server) Create issue templates for baby-project

### PR DESCRIPTION
**Pull request by dusan-turajlic**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Detailed Description
+<!--- Provide a detailed description of the change or addition you are proposing -->
+
+## Context
+<!--- Why is this change important to you? How would you use it? -->
+<!--- How can it benefit other users? -->
+
+## Possible Implementation
+<!--- Not obligatory, but suggest an idea for implementing addition or change -->
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
<!--- Optional. -->
<!--- Suggest an idea for implementing the proposed change or addition. -->
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+title: "(Feature) Short description here"
+labels: enhancement
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Detailed Description
+<!--- Provide a detailed description of the change or addition you are proposing -->
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
<!--- Provide a detailed description of the change or addition you are proposing. -->
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: "(Feature) Short description here"
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/feature_request.md]:
```suggestion
title: "(Feature) General summary of the issue"
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+labels: enhancement
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/feature_request.md]: This comment seems unnecessary if we say what to add in the actual title.
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+name: Bug
+about: Create a report to help us improve
+title: "(BUG) Short description here"
+labels: bug, help wanted, question
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/bug.md]: This comment seems unnecessary if we say what to add in the actual title.
```suggestion
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
+---
+name: Bug
+about: Create a report to help us improve
+title: "(BUG) Short description here"
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
title: "(BUG) General summary of the issue"
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
+---
+name: Bug
+about: Create a report to help us improve
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
about: Create a bug report to help us improve this project
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Description
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
<!--- Tell us what currently happens. -->
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Description
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
+
+## Expected Behavior
+<!--- Tell us what should happen -->
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
<!--- Tell us what should happen. -->
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 10]
```diff
@@ -0,0 +1,33 @@
...
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
+
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
+
+## Steps to Reproduce
+<!--- Provide a link to a live example, or an unambiguous set of steps to -->
+<!--- reproduce this bug. Include code to reproduce, if relevant -->
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/bug.md:24]:
```suggestion
<!--- reproduce this bug. Include code to reproduce, if relevant. -->
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+## Detailed Description
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/feature_request.md:10]:
```suggestion
## Description
```

**Review by AndreasNasman — APPROVED** [review]
I think the order of the headings should be as consistent as possible between the two templates.

| Bug report | Feature request |
| ------------- | ------------- |
| Description | Description |
| Context | Context |
| Expected Behavior | |
| Actual Behavior | |
| Steps to Reproduce | |
| Possible Fix | Possible Implementation |

File: .github/ISSUE_TEMPLATE/bug.md [hunk 12]
```diff
@@ -0,0 +1,33 @@
...
+title: "(BUG) Short description here"
+labels: bug, help wanted, question
+assignees: ''
+
+---
+
+<!--- Provide a general summary of the issue in the Title above -->
+
+## Description
+<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug -->
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
<!--- Provide a more detailed introduction to the issue itself, and why you consider it to be a bug. -->
```

File: .github/ISSUE_TEMPLATE/bug.md [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+
+## Expected Behavior
+<!--- Tell us what should happen -->
+
+## Actual Behavior
+<!--- Tell us what happens instead -->
+
+## Possible Fix
+<!--- Not obligatory, but suggest a fix or reason for the bug -->
```
Comment (AndreasNasman) [inline .github/ISSUE_TEMPLATE/bug.md]:
```suggestion
<!--- Optional. -->
<!--- Suggest a fix or reason for the bug. -->
```

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 11 above)
Comment (dusan-turajlic) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: Actualy I think we whould keep the Detailed thing here. Its a project that might get more than the just us here so clarifytin "Detailed" might be a good idea

Comment (AndreasNasman) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: We can keep it if you want as long as it's consistent among the templates; the other one has just **Description** at the moment.

**Comment by AndreasNasman** [comment]
Typo in 'templates' in commit message and issue title btw 👆
`(baby-project-server) Create issue templates for baby-project`

File: .github/ISSUE_TEMPLATE/feature_request.md (hunk 11 above)
Comment (dusan-turajlic) [reply .github/ISSUE_TEMPLATE/feature_request.md:10]: Good Point, I add that there as well. I used just stock temolate from here https://github.com/TalAter/open-source-templates

**Pull request merged**